zerver/lib/zcommand.py

```diff
@@ -54,4 +58,47 @@ def change_mode_setting(command: str, switch_command: str,
                                             switch_command='fluid-width',
                                             setting='fluid_layout_width',
                                             setting_value=False))
+    elif command == 'digress':
+        data_keys = [
+            "old_stream",
+            "old_topic",
+            "new_stream",
+            "new_topic",
+        ]
+        if not data or not all(key in data for key in data_keys):
+            raise JsonableError(_('Invalid data.'))
```

Comment:
Also, inform user what is the correct use of /digress command. Maybe with an example.